package.json

```diff
@@ -13,24 +13,30 @@
   ],
   "scripts": {
     "test": "mocha",
-    "prepublish": "run-s clean trie:** rollup:**",
+    "prepublish": "run-s clean trie:** rollup",
     "trie:data": "babel-node src/opentype/shapers/generate-data.js",
     "trie:use": "babel-node src/opentype/shapers/gen-use.js",
     "trie:indic": "babel-node src/opentype/shapers/gen-indic.js",
     "trie:copy": "shx cp src/opentype/shapers/*.trie ./",
-    "rollup:index": "rollup -c -m -i src/index.js -o index.js",
-    "rollup:base": "rollup -c -m -i src/base.js -o base.js",
+    "rollup": "rollup -c",
     "clean": "shx rm -f index.js base.js data.trie indic.trie use.trie src/opentype/shapers/data.trie src/opentype/shapers/use.trie src/opentype/shapers/use.json src/opentype/shapers/indic.trie src/opentype/shapers/indic.json",
     "coverage": "cross-env BABEL_ENV=cover nyc mocha"
   },
   "main": "index.js",
   "jsnext:main": "src/index.js",
+  "browser": "index.browser.js",
   "files": [
     "src",
     "base.js",
     "base.js.map",
     "index.js",
     "index.js.map",
+    "base.es5.js",
+    "base.es5.js.map",
+    "index.es5.js",
+    "index.es5.js.map",
```

Comment:
So these ES5 versions wouldn't be accessible through package.json, you'd have to manually reference them like `import fontkit from 'fontkit/index.es5.js'`? Not sure that's going to be obvious...

Many bundlers (e.g. webpack, rollup) don't compile things in node_modules, so the default would still be not supporting IE11. Seems bad...

Reply:
> So these ES5 versions wouldn't be accessible through package.json, you'd have to manually reference them like import fontkit from 'fontkit/index.es5.js'

Yes you are right. Its a breaking change

> Many bundlers (e.g. webpack, rollup) don't compile things in node_modules, so the default would still be not supporting IE11

I can point the browser entry to point es5 version but won't help much those devs still supporting IE11 because many of the dependencies uses ES6 syntax nowadays. Le me know what should i do.

Reply:
> many of the dependencies uses ES6 syntax nowadays

Which ones?

Reply:
> Which ones?

restructure, dfa, unicode-trie, unicode-properties

Reply:
Regarding IE11 support, many libraries are dropping out of box support for it due to its usage decline. Even Microsoft is actively promoting to developers/users drop support for it.

By distributing by default IE11 compatible build, its penalizing performance wise majority of users that uses modern browsers. It also increase code size considerably.

Another point is that makes debugging / profiling harder. Sometime ago i tried profiling pdf generation by pdfkit and i gave up because the profile trace was replete of artifacts introduced by transpilation / core-js.

Here's some more info about IE11:  https://css-tricks.com/a-business-case-for-dropping-internet-explorer/

PS: https://fontkit-demo.now.sh/ and http://pdfkit.org/demo/browser.html are already incompatible with IE 11 (and no one complained until now)